CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ## Current Main 
 
+### Bug Fixes
+
+- fixed invalid example GeoJSON in swagger ([#835])
```

Comment:
We could reference the issue itself here (#831) rather than the PR.
Or do you see an advantage in using the PR?
The issue is a more permanent thing compared to the PR.